app/modules/ads/bill-the-lizard-wrapper.js

```diff
@@ -109,6 +109,8 @@ export const billTheLizardWrapper = {
         slotService.disable(getNextIncontentId(), 'catlapsed');
 
         utils.logger(logGroup, `catlapsing ${getNextIncontentId()}`);
+        // eslint-disable-next-line no-console
+        console.log(`catlapsing ${getNextIncontentId()}`);
```

Comment:
We have a deal! If we use `catlapsing ${getNextIncontentId()}` in one more place we'll assign it to a variable which we'll reuse ;)

Reply:
Hi! Why would we want console.logs on production? :)

Reply:
Good question and my bad! I've checked how we do it in "a kosher way" and we can improve it most likely and do consistently: https://github.com/Wikia/mobile-wiki/blob/501663b1715a19d72d2b745a5308883e2dcbd354/app/components/ad-slot.js#L55

Reply:
I had no idea such service exists, my bad :/